### ISSUE TITLE
i did Fine-tuning model with facenet But ...

### ISSUE THREAD

**Issue by NguyenThaiHoc1** [davidsandberg/facenet#985]
I did pretrained facenet model. But I received information of checkpoint of epochs. and I don't know that loss is very low (In the picture). I pretrained on my dataset which have range 400 classes

I used it to predict, result is very bad 

I need everyone help to resolve this my problem 

![image](https://user-images.githubusercontent.com/22360388/53477795-0c7dd280-3aa8-11e9-8fe4-b92b99d5bca7.png)

Some information
I get the number of layer 449
![image](https://user-images.githubusercontent.com/22360388/53478222-e86ec100-3aa8-11e9-97d7-ef26aacaafcb.png)
layer.trainable to pretrained 

I use new version checkpoint 20180408 

Thanks you for reading 
Sorry for my English is bad
